tsfresh/feature_extraction/settings.py

```diff
@@ -117,7 +119,7 @@ def __init__(self):
             "ar_coefficient": [{"coeff": coeff, "k": k} for coeff in range(5) for k in [10]],
             "mean_abs_change_quantiles": [{"ql": ql, "qh": qh}
                                           for ql in [0., .2, .4, .6, .8] for qh in [.2, .4, .6, .8, 1.]],
-            "fft_coefficient": [{"coeff": coeff} for coeff in range(0, 10)],
+            "fft_coefficient": [{"coeff": k, "attr": a} for a, k in product(["real", "imag"], range(100))],
```

Comment:
It only depends on the length of the time series. For a time series of length 10, you will get exactly 10 dft coefficients. For one of length 100, exactly 100